site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Community Plugins in the Marketplace
+subsection: Plugins (Beta)
+weight: 60
+---
+
+Once your plugins has reached a certain level of quality, you might consider submitting it to the Plugin Marketplace. The Plugin Marketplace allows discovery, installation and updates of plugins directly within Mattermost. It is a great way to get feedback on your plugin and help making it more popular. Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
+
+### Requirements for adding community plugin to the Marketplace
+Every community plugin must fulfill the following checklist to be added to the Marketplace:
+
+**Product Requirements (Checked by a Product Manager)**
+
+1. The plugin is published under an Apache v2 compatible license (e.g. no GPL, APGL). A list of compatible licenses can be found [here](https://apache.org/legal/resolved.html#category-a).
+2. The source code is available in a public git repository.
+3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
+4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features and a development guide. This is typically a README file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [README of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
+5. For every release a changelog has to be publish. The link to it is noted via `release_notes_url` in the manifest.
+6. The plugin has to be out of Beta and be released with at least v1.0.0.
+7. All configuration has to be possible using the UI of Mattermost.
+8. The plugin id defined in the manifest must not collide with the id of an existing plugin in the marketplace. It should follow [the naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
+
+**Technical Requirements (Checked by a developers of the Toolkit or the Integrations team)**
+
+1. The plugin works for 60k concurrent connections and in a high availability environment. (There are currently no tools available to verify this property. Hence, it is checked via code review by a developer)
+2. The plugin logs important events on appropriate log levels to allow system admins to troubleshoot issues.
+
+**Security Requirements (Checked by a member of the security team)**
+
+1. The plugin does not expose a vulnerability.
+2. The plugin does not include favor the author of the plugin or a third party excessively by e.g. including a bitcoin miner that mines on behalf of the author.
+3. The plugins author must notify Mattermost about any vulnerabilities in the future.
```

Comment:
nit: this is a future requirement, not one that can be checked before adding to the marketplace. Do we want to set communication expectations here instead? (i.e. you must register an email address for communication with the security team)